tensorflow/python/keras/optimizer_v2/adadelta.py

```diff
@@ -57,7 +57,8 @@ class Adadelta(optimizer_v2.OptimizerV2):
       ([pdf](http://arxiv.org/pdf/1212.5701v1.pdf))
 
   """
-
+  #def __init__(self, lr=1.0, rho=0.95, epsilon=None, decay=0., **kwargs):
+  #Adadelta function definition as per paper by M.D. Zeiler (https://arxiv.org/pdf/1212.5701.pdf) where epsilon=1e-6and learning rate=1.0
```

Comment:
Please put this at top of docstring.
Also please put description, i.e., which section in this paper that a recommended learning rate is proposed so users are well aware of it.

Reply:
Hi @tanzhenyu, sure I will put it at the top of the docstring.